sklearn/utils/tests/test_linear_assignment.py

```diff
@@ -1,13 +1,16 @@
 # Author: Brian M. Clapper, G Varoquaux
 # License: BSD
 
-import numpy as np
+# TODO: Remove this test module as the methods being tested
```

Comment:
Please add a line above with `# 0.23` so we know when to remove this file

Reply:
done